.github/workflows/SpellCheck.yml

```
@@ -0,0 +1,13 @@
+name: Spell Check
+
+on: [pull_request]
```

Also allow running manually?
```suggestion
on: [pull_request, workflow_dispatch]
```